Detectors/FIT/FT0/workflow/src/FT0DigitWriterSpec.cxx

```diff
@@ -0,0 +1,46 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   FT0DigitWriterSpec.cxx
+
+#include <vector>
+
+#include "DPLUtils/MakeRootTreeWriterSpec.h"
+#include "DataFormatsFT0/ChannelData.h"
+#include "DataFormatsFT0/Digit.h"
+#include "FT0Workflow/FT0DigitWriterSpec.h"
+using namespace o2::framework;
+
+namespace o2
+{
+namespace ft0
+{
+
+template <typename T>
+using BranchDefinition = MakeRootTreeWriterSpec::BranchDefinition<T>;
+DataProcessorSpec getFT0DigitWriterSpec()
```

Comment:
There is an existing FT0DigitWriterSpec in `Steer/DigitizerWorkflow`.
I agree that we should move detector specific components away from there. So could you make the transition proper and migrate completely?

Reply:
Hello
I have 2 points:
1. I can not open  log of clang errors
2. Its not really transition, just conversion raw data to digits  for simplifying next QC and calibration.